model/src/test/kotlin/AdvisorRecordTest.kt

```diff
@@ -36,30 +36,22 @@ import org.ossreviewtoolkit.utils.common.enumSetOf
 class AdvisorRecordTest : WordSpec({
     "hasIssues" should {
         "return false given the record has no issues" {
-            val vul1 = createVulnerability("CVE-2021-1")
-            val vul2 = createVulnerability("CVE-2021-2")
             val record = AdvisorRecord(
                 sortedMapOf(
-                    langId to listOf(createResult(1)),
-                    queryId to listOf(createResult(1, vulnerabilities = listOf(vul1, vul2)))
+                    langId to listOf(createResult(1))
```

Comment:
Commit message:
- Spell "misleading" as one word.
- "This makes it" -> maybe "That makes this test"